components/OverviewCard/index.js

```diff
@@ -22,7 +22,7 @@ const StyledProjectCard = styled.div`
   flex-direction: column;
   align-items: center;
   justify-content: center;
-  border: 1px solid black;
+  border: 1px solid rgb(var(--foreground-rgb));
   border-radius: 10px;
   padding: 10px;
   margin: 10px;
```

Comment:
I'm wondering whether here it would make sense to choose a more responsive unit like em or rem? But I'm not sure, also in how far you already consider how your styling looks on different screens....

Reply:
it is only a draft styling not the final, i will do it in a seperate user story